eldev-util.el

```diff
@@ -1483,5 +1482,7 @@ absolute paths, but are traced without them.  See
           (eldev--do-path-matches actual-path pattern-path-rest)
           (and actual-path-rest (eldev--do-path-matches actual-path-rest pattern-path))))))
 
+(defmacro eldev--is-windows-nt ()
+  (eq system-type 'windows-nt))
```

Comment:
I suggest using a plain function and conform to elisp nameing conventions:

```elisp
(defun eldev--windows-nt-p ()
  (eq system-type 'windows-nt))
```

Reply:
Very good point indeed!

(I have since replaced that macro with `eldev-util.el:eldev--shell-script-name`, I had misunderstood the feedback which form to factor into a fn)